vendor/vendor.json

```diff
@@ -47,5 +53,5 @@
 			"revision": "d627af88ac021317a280d320986282a6dc97549a"
 		}
 	],
-	"rootPath": "github.com/heroku/go-getting-started"
+	"rootPath": "github.com/Xe/go-getting-started"
```

Comment:
Revert this line. I suggest checking out forks into the original repo as a remote commit.